doc/examples.md

```diff
@@ -1,154 +1,184 @@
-# Examples
+# How-tos
```

Comment:
I wonder if we should split out the differnt "How-to guides" into separate files.. but this will lead to plenty of tiny files, which are hard to maintain. But it would be nice if we could somehow link each individual Howto in the pimary navigation, similar to "Use DBus config API" and "Desktop integration".